src/test/java/com/puppycrawl/tools/checkstyle/XpathFileGeneratorAuditListenerTest.java

```diff
@@ -61,6 +62,14 @@
     private static final LocalizedMessage FOURTH_MESSAGE = createLocalizedMessage(5, 5,
             TokenTypes.VARIABLE_DEF, "JavadocModuleId", JavadocVariableCheck.class);
 
+    /**
+     * Output stream to hold the test results. The IntelliJ IDEA issues the AutoCloseableResource
+     * warning here, so it need to be suppressed. The {@code ByteArrayOutputStream} does not hold
```

Comment:
I do not see suppression for IDEA inspection, I do not see TC failure, please explain this comment.

Reply:
Copy pasted from `XMLLoggerTest`, 
removed comment